data/mdn

```diff
@@ -1,5 +1,6 @@
 developer.allizom.org
 developer.mozilla.org
+mdn.dev
 mdn.mozit.cloud
```

Comment:
@yin1999 FYI

- We will stop using `mdn.mozit.cloud` soon.
- We stopped using `mdn.mozillademos.org` (it redirects to `mdn.dev`).
- You could add `mdn.mozilla.net` and `mdn.allizom.net`, because there's also `live-samples.mdn.mozilla/allizom.net` (`allizom.net` is rather new).